Pully Penelope/Assets/Scripts/PlayerMovement.cs

```diff
@@ -22,11 +30,37 @@ private void Update()
         positionChange = Vector3.zero;
         positionChange.x = Input.GetAxisRaw("Horizontal");
         positionChange.y = Input.GetAxisRaw("Vertical");
-
-        /*if (positionChange != Vector3.zero)
+        grabFloat = Input.GetAxisRaw("Jump");
+        if (handleName != null && grabFloat != 0)
         {
-            MovePlayer();
-        }*/
+            if (handleName == "DHandle")
+            {
+                handleOrientation.x = 0;
+                positionChange.x = handleOrientation.x;
+                handleOrientation.y = 1;
+            }
+            else if (handleName == "UHandle")
+            {
+                handleOrientation.x = 0;
+                positionChange.x = handleOrientation.x;
+                handleOrientation.y = -1;
+            }
+            else if (handleName == "RHandle")
+            {
+                handleOrientation.x = -1;
+                handleOrientation.y = 0;
+                positionChange.y = handleOrientation.y;
+            }
+            else if (handleName == "LHandle")
+            {
+                handleOrientation.x = 1;
+                handleOrientation.y = 0;
+                positionChange.y = handleOrientation.y;
+            }
```

Comment:
You are checking one variable for different values, so this would be a good situation to use a switch statement instead of a ton of ifs/else ifs.